app/models/early_career_payments/eligibility.rb

```diff
@@ -169,6 +169,12 @@ class Eligibility < ApplicationRecord
       AcademicYear.new => AcademicYear::Type.new.serialize(AcademicYear.new)
     }
 
+    @@max_award_amount_in_pounds = AWARD_AMOUNTS.collect(&:uplift_amount).max
```

Comment:
Are you ok with these class valiables? What about:

```ruby
@max_award_amount_in_pounds = AWARD_AMOUNTS.collect(&:uplift_amount).max

class << self
  attr_accessor :max_award_amount_in_pounds
end
```

Reply:
After I wrote this comment I saw the `award.rb` file and it seems you're a fan of using `@@`. If these classes don't get inherited I guess it's fine.

Reply:
I totally know what you mean. I cracked open a Ruby textbook about the whole `@@` and class inheritance thing. This is somewhat related to the massive structure which will eventually hold the mapping from URN (the school identifier) to award amount. I definitely won't want multiple copies of that hanging about. I was hoping to kick the `@@` decision down the road until I work on the next story.